tests/php82_readonly_class.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+readonly class support in php 8.2+
+--SKIPIF--
+<?php if (!class_exists('AllowDynamicProperties')) die('skip PHP >=8.2 only'); ?>
```

Comment:
I'd prefer checking PHP_VERSION_ID here.

Reply:
This was copied from php82_metadata.phpt and I missed that - I believe that was originally written months ago while the AllowDynamicProperties class wasn't in a published release yet.

I changed both uses